src/org/labkey/snd/SNDUserSchema.java

```diff
@@ -138,51 +139,31 @@ public TableInfo createTable(SNDUserSchema schema, ContainerFilter cf)
                     @Override
                     public TableInfo createTable(SNDUserSchema schema, ContainerFilter cf)
                     {
-                        if (!schema.getPermissionCheck() || schema.getContainer().hasPermission(schema.getUser(), AdminPermission.class))
-                        {
-                            return new EventNotesTable(schema, SNDSchema.getInstance().getTableInfoEventNotes(), cf).init();
-                        }
-
-                        return null;
+                        return new EventNotesTable(schema, SNDSchema.getInstance().getTableInfoEventNotes(), cf, schema.getContextualRole()).init();
```

Comment:
Don't really need to do all this work to propagate the contextual role if no one ever sets it except the SNDUserSchema.  (the table can just ask the schema)

Reply:
Wasn't sure about further work here. Moved this up to user schema as suggested